disnake/message.py

```diff
@@ -1199,6 +1203,15 @@ def jump_url(self) -> str:
         guild_id = getattr(self.guild, "id", "@me")
         return f"https://discord.com/channels/{guild_id}/{self.channel.id}/{self.id}"
 
+    @property
```

Comment:
shouldn't it be a cached property?

Reply:
It's possible to delete a thread and create a new thread in which case the cached thread would point to the wrong object.